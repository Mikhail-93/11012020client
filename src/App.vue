<template>
    <v-app id="inspire">
        <v-navigation-drawer v-model="drawer" app>
            <v-list dense>
                <v-list-item link>
                    <v-list-item-action>
                        <v-icon>mdi-home</v-icon>
                    </v-list-item-action>
                    <v-list-item-content>
                        <v-list-item-title>Home</v-list-item-title>
                    </v-list-item-content>
                </v-list-item>
                <v-list-item link>
                    <v-list-item-action>
                        <v-icon>mdi-contact-mail</v-icon>
                    </v-list-item-action>
                    <v-list-item-content>
                        <v-list-item-title>Contact</v-list-item-title>
                    </v-list-item-content>
                </v-list-item>
            </v-list>
        </v-navigation-drawer>

        <v-app-bar
                app
                color="indigo"
                dark
        >
            <v-app-bar-nav-icon @click.stop="drawer = !drawer"/>
            <v-toolbar-title>Application</v-toolbar-title>
        </v-app-bar>

        <v-content>
            <v-container class="fill-height pa-0" fluid>
                <LMap ref="map" style="z-index: 0">
                    <l-tile-layer
                            url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
                            attribution='Map data: &copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>, <a href="http://viewfinderpanoramas.org">SRTM</a> | Map style: &copy; <a href="https://opentopomap.org">OpenTopoMap</a> (<a href="https://creativecommons.org/licenses/by-sa/3.0/">CC-BY-SA</a>)'
                    >
                    </l-tile-layer>
                    <!--                    <l-control-attribution position="bottomright" prefix="leaflet OpenStreetMap Contributions" >-->

                </LMap>
                <!--                class="align-stretch"-->
            </v-container>
        </v-content>
    </v-app>
</template>

<script>
    import {LMap, LTileLayer, LMarker, LControlAttribution} from 'vue2-leaflet';

    export default {
        components: {
            LMap,
            LTileLayer,
            LMarker,
            LControlAttribution
        },
        props: {
            source: String,
        },
        data: () => ({
            drawer: null,
        }),
        mounted() {
            this.$nextTick(() => {
                // this.map=
                this.$refs.map.mapObject.setView([51.505, -0.09], 13)

            })
        }
    }
</script>