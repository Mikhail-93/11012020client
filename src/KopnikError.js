class KopnikError extends Error {
    constructor(message, code) {
        super(message)
        this.code = code
    }
}
class KopnikApiError extends KopnikError {
    constructor(message, code, url) {
        super(message, code)
        this.url = url
    }
}



export {KopnikError, KopnikApiError}
