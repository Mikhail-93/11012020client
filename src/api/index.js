import 'isomorphic-fetch'
import fetchIntercept from "fetch-intercept/lib/node";

import errorInterceptor from "./errorInterceptor";
import dataInterceptor from "./dataInterceptor";
import cookieInterceptor from "./cookieInterceptor";
import headersInterceptor from "./headersInterceptor";
import api from "./api";

fetchIntercept.register(errorInterceptor)
fetchIntercept.register(dataInterceptor)
fetchIntercept.register(cookieInterceptor)
fetchIntercept.register(headersInterceptor)



export default api
