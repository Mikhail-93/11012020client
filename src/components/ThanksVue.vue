<template>
    <v-container fluid class="fill-height">
        <v-card v-for="(eachThank, eachThankIndex) in value" :key="eachThankIndex"
                elevation="12" class="mb-10 mx-auto" width="100%" max-width="350px">
            <kopnik-view :value="eachThank.who"></kopnik-view>
            <v-divider></v-divider>
            <v-list>
                <v-list-item v-for="(eachDeal, eachDealIndex) of eachThank.deals" :key="eachDealIndex">
                    <v-list-item-content>
                        <v-list-item-title :title="eachDeal.project">{{eachDeal.project}}</v-list-item-title>
                        <v-list-item-subtitle>{{eachDeal.role}}</v-list-item-subtitle>
                        <v-list-item-subtitle>{{eachDeal.results}}</v-list-item-subtitle>
                    </v-list-item-content>
                    <v-list-item-action v-if="eachDeal.link">
                        <a :href="eachDeal.link" target="_blank" style="text-decoration: inherit">
                            <v-btn icon @click="">
                                <v-icon>mdi-link</v-icon>
                            </v-btn>
                        </a>
                    </v-list-item-action>
                </v-list-item>
            </v-list>
        </v-card>

        <div style="width: 100%"></div>

        <v-card elevation="12" class="mx-auto" width="100%" max-width="350px">
                <v-img src="https://technicalsupportoverseas.com/wp-content/uploads/revslider/AboutUs/slider-rev-about.png"></v-img>
            <v-card-title>
                Тоже хочешь участвовать?
            </v-card-title>
            <v-card-subtitle>
                Принять участие в создании первой международной сети Копного права очень просто. Узнай как присоединиться.
            </v-card-subtitle>
            <v-card-actions>
                <v-btn color="primary" @click="join_click" class="mx-auto">
                    Узнать как присоединиться
                </v-btn>
            </v-card-actions>
        </v-card>
    </v-container>
</template>
<script>
    import Kopnik from "../models/Kopnik"
    import thanks from "../thanks";

    import KopnikView from './KopnikVue'
    import logger from "./mixin/logger";
    import {container} from "../plugins/bottle";

    export default {
        name: "Thanks",
        mixins: [logger],
        components: {
            KopnikView
        },
        data: () => {
            return {
                application: container.application,
                value: [],
            }
        },
        props: {},
        computed: {},
        watch: {},
        methods: {
            join_click() {
                window.open('https://docs.google.com/document/d/1NzlfhHoDkT9FBR1aH41bZZ_eZQeUH7kTGTXABpvY3YE/edit#heading=h.pn4ly25488lj')
            }
        },
        async created() {
            await this.application.resolveUser()
            this.value = thanks.map(eachThank => {
                let eachWho
                eachWho = new Kopnik()
                eachWho.id = eachThank.who.id
                eachWho.merge(eachThank.who)
                if (!eachThank.who.id || !this.application.user) {
                    eachWho.isLoaded = true
                }
                eachWho.firstName = eachThank.who.name
                eachThank.who = eachWho
                return eachThank
            })
        },
    }
</script>
