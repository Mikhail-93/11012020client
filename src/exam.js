const cars = {};Debugger attached.
const carHonda = {model: 'CR-V', year: 2015};
const carVolvo = {model: 'XC-60', year: 2018};
const carBMW = {model: 'X5', year: 2019};
cars[carHonda] = 'Good';
cars[carVolvo] = 'Better';
cars[carBMW] = 'The best';
console.log('Honda is', cars[carHonda]);
console.log('Volvo is', cars[carVolvo]);
console.log('BMW is', cars[carBMW]);
