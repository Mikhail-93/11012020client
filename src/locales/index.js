import ru from './ru'
import en from './en'
import sk from './sk.json'

const messages = {
  ru,
  en,
  sk,
}

export default messages
