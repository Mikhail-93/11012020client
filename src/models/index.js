/**
 * Created by alexey2baranov on 8/20/16.
 */

import AbstractSync from "./AbstractSync" //для того чтобы правильно работали рекурсивные ссылки модулей
// import Registration from "./model/Registration"
import Kopnik from "./Kopnik"
// import Kopa from "./model/Kopa"
// import File from "./model/File"

export {AbstractSync, Kopnik}