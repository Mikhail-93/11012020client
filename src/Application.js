import {AbstractSync, Kopnik} from "./models";

export default class Application {
    static getInstance() {
        if (!this.instance) {
            this.instance = new this()
        }
        return this.instance
    }

    user = null

    constructor() {
        /**
         * Кэш моделей
         * @type {Array}
         */
        // пока не понятно зачем приложению кэш.
        // он вероятно будет сильно торможить приложение из-за реактивности App.$data в которой он находится
        // хотя он же не в App.$data а в App.$options !!
        // уже в дата потому тчо в options не реактивен
        //this.models = AbstractSync.cache

        /**
         * Идентификатор раздела
         * Равен названиею соответствующего компонента
         *
         * @type {string} Map | Profile | Thanks
         */
        this.SECTION = "Map"

        this.user = undefined

        this.initUser()
    }

    getSharedState() {
        return {
            SECTION: this.SECTION
        }
    }

    setState(state) {
        if (state.SECTION) {
            this.SECTION = state.SECTION
        }
    }

    /**
     * Инициализирует ползователя при запуске приложения
     *
     * @returns {Promise<void>}
     */
    async initUser() {
        let vkUser = localStorage.getItem("vkUser")
        if (vkUser) {
            vkUser = JSON.parse(vkUser)
            let user //= await Kopnik.getVkUserStatus(vkUser.uid)
            if (vkUserStatus) {
                this.user= Kopnik.merge(vkUserStatus.user)
            } else {
                this.setVkUser(vkUser)
                this.SECTION="Profile"
            }
        }
    }

    /**
     * Учстанавливает пользователем объект из VK.init::success()
     * @param vkUser
     * @returns
     */
    setVkUser(vkUser){
        this.user= new Kopnik
        this.user.merge({
            hash: vkUser.hash,
            uid: vkUser.uid,
            firstname: vkUser.first_name,
            surname: vkUser.last_name,
            photo: vkUser.photo,
            smallPhoto: vkUser.photo_rec,
        })
        localStorage.setItem("vkUser", JSON.stringify(vkUser))
    }
}
