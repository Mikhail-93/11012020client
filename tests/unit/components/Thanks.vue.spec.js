import 'isomorphic-fetch'
import {mount} from '@vue/test-utils'

import {Kopnik} from "../../../src/models";
import flushPromises from "flush-promises";
import '../../vue-setup'
import ThanksVue from "../../../src/components/ThanksVue";

describe('Thanks', () => {
    it('renders', async () => {
        await Promise.all([1,2,3].map(eachId=>Kopnik.get(eachId)))
        const wrapper = mount(ThanksVue)
        expect(wrapper.vm.$el).toMatchSnapshot()
    })
})
