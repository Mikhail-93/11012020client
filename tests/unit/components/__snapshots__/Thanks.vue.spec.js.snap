// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thanks renders 1`] = `
<div
  class="flex xs11 md6 xl4 mx-auto"
>
  <div
    class="mb-10 v-card v-sheet theme--light elevation-12"
  >
    <div
      class="v-list v-sheet v-sheet--tile theme--light"
      role="list"
    >
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-avatar v-list-item__avatar {avatarMxAuto: 'mx-auto'}"
          style="height: 150px; min-width: 150px; width: 150px;"
        >
          <img
            src="/avatar.png"
            style="object-fit: cover;"
          />
        </div>
      </div>
       
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-list-item__content"
        >
          <div
            class="v-list-item__title title"
          >
            
                Арт Арт Арт
            
          </div>
        </div>
      </div>
       
      <!---->
       
      <!---->
       
      <!---->
    </div>
     
    <hr
      aria-orientation="horizontal"
      class="v-divider theme--light"
      role="separator"
    />
     
    <div
      class="v-list v-sheet v-sheet--tile theme--light"
      role="list"
    >
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-list-item__content"
        >
          <div
            class="v-list-item__title"
          >
            kopnik.org v0.1
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            Бэкенд
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="mb-10 v-card v-sheet theme--light elevation-12"
  >
    <div
      class="v-list v-sheet v-sheet--tile theme--light"
      role="list"
    >
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-avatar v-list-item__avatar {avatarMxAuto: 'mx-auto'}"
          style="height: 150px; min-width: 150px; width: 150px;"
        >
          <img
            src="/avatar.png"
            style="object-fit: cover;"
          />
        </div>
      </div>
       
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-list-item__content"
        >
          <div
            class="v-list-item__title title"
          >
            
                Баранов Алексей qwer
            
          </div>
        </div>
      </div>
       
      <!---->
       
      <!---->
       
      <!---->
    </div>
     
    <hr
      aria-orientation="horizontal"
      class="v-divider theme--light"
      role="separator"
    />
     
    <div
      class="v-list v-sheet v-sheet--tile theme--light"
      role="list"
    >
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-list-item__content"
        >
          <div
            class="v-list-item__title"
          >
            kopnik.org v0.1
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            Web-фронтенд/Тимлид
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            
          </div>
        </div>
      </div>
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-list-item__content"
        >
          <div
            class="v-list-item__title"
          >
            Макет сайта kopnik.org
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            Автор
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            
          </div>
        </div>
      </div>
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-list-item__content"
        >
          <div
            class="v-list-item__title"
          >
            Правила поведения на улице kopnik.org
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            Сходатай на копе
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            
          </div>
        </div>
      </div>
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-list-item__content"
        >
          <div
            class="v-list-item__title"
          >
            Устав команды по созданию kopnik.org
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            Автор
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            
          </div>
        </div>
      </div>
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-list-item__content"
        >
          <div
            class="v-list-item__title"
          >
            Техническое задание на создание kopnik.org
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            Заказчик/Архитектор/Техписатель
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            
          </div>
        </div>
      </div>
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-list-item__content"
        >
          <div
            class="v-list-item__title"
          >
            Расчет и обоснование стоимости создания kopnik.org
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            Заказчик/Архитектор/Техписатель
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="mb-10 v-card v-sheet theme--light elevation-12"
  >
    <div
      class="v-list v-sheet v-sheet--tile theme--light"
      role="list"
    >
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-avatar v-list-item__avatar {avatarMxAuto: 'mx-auto'}"
          style="height: 150px; min-width: 150px; width: 150px;"
        >
          <img
            style="object-fit: cover;"
          />
        </div>
      </div>
       
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-list-item__content"
        >
          <div
            class="v-list-item__title title"
          >
            
                 Игорь 
            
          </div>
        </div>
      </div>
       
      <!---->
       
      <!---->
       
      <!---->
    </div>
     
    <hr
      aria-orientation="horizontal"
      class="v-divider theme--light"
      role="separator"
    />
     
    <div
      class="v-list v-sheet v-sheet--tile theme--light"
      role="list"
    >
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-list-item__content"
        >
          <div
            class="v-list-item__title"
          >
            kopnik.org v0.1
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            Системное администрирование
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            
          </div>
        </div>
      </div>
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-list-item__content"
        >
          <div
            class="v-list-item__title"
          >
            Сервера kopnik.org и dev.kopnik.org
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            Хозяин
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="mb-10 v-card v-sheet theme--light elevation-12"
  >
    <div
      class="v-list v-sheet v-sheet--tile theme--light"
      role="list"
    >
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-avatar v-list-item__avatar {avatarMxAuto: 'mx-auto'}"
          style="height: 150px; min-width: 150px; width: 150px;"
        >
          <img
            src="https://sun1-21.userapi.com/iapk22EQe5H-gyZdPrmmWz7n77nzOwuNG3AqOw/j0EX7sM_ZCE.jpg"
            style="object-fit: cover;"
          />
        </div>
      </div>
       
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-list-item__content"
        >
          <div
            class="v-list-item__title title"
          >
            
                 Евгений Нигматуллинн 
            
          </div>
        </div>
      </div>
       
      <!---->
       
      <!---->
       
      <!---->
    </div>
     
    <hr
      aria-orientation="horizontal"
      class="v-divider theme--light"
      role="separator"
    />
     
    <div
      class="v-list v-sheet v-sheet--tile theme--light"
      role="list"
    >
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-list-item__content"
        >
          <div
            class="v-list-item__title"
          >
            Новый временный сайт kopnik.org
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            Веб-мастер
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            
          </div>
        </div>
      </div>
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-list-item__content"
        >
          <div
            class="v-list-item__title"
          >
            Разделы "Нас поддерживают" и "Мы в соцсетях" на временном сайте kopnik.org
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            Веб-мастер
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="mb-10 v-card v-sheet theme--light elevation-12"
  >
    <div
      class="v-list v-sheet v-sheet--tile theme--light"
      role="list"
    >
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-avatar v-list-item__avatar {avatarMxAuto: 'mx-auto'}"
          style="height: 150px; min-width: 150px; width: 150px;"
        >
          <img
            src="https://sun9-43.userapi.com/c849124/v849124535/9890d/jqIsFAEYLdA.jpg"
            style="object-fit: cover;"
          />
        </div>
      </div>
       
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-list-item__content"
        >
          <div
            class="v-list-item__title title"
          >
            
                 Дмитрий Журбин 
            
          </div>
        </div>
      </div>
       
      <!---->
       
      <!---->
       
      <!---->
    </div>
     
    <hr
      aria-orientation="horizontal"
      class="v-divider theme--light"
      role="separator"
    />
     
    <div
      class="v-list v-sheet v-sheet--tile theme--light"
      role="list"
    >
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-list-item__content"
        >
          <div
            class="v-list-item__title"
          >
            Перевод ролика о сборе средств на создание kopnik.org
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            Переводчик/Голос/Монтажер
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="mb-10 v-card v-sheet theme--light elevation-12"
  >
    <div
      class="v-list v-sheet v-sheet--tile theme--light"
      role="list"
    >
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-avatar v-list-item__avatar {avatarMxAuto: 'mx-auto'}"
          style="height: 150px; min-width: 150px; width: 150px;"
        >
          <img
            src="/avatar.png"
            style="object-fit: cover;"
          />
        </div>
      </div>
       
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-list-item__content"
        >
          <div
            class="v-list-item__title title"
          >
            
                Титов Александр Николаевич
            
          </div>
        </div>
      </div>
       
      <!---->
       
      <!---->
       
      <!---->
    </div>
     
    <hr
      aria-orientation="horizontal"
      class="v-divider theme--light"
      role="separator"
    />
     
    <div
      class="v-list v-sheet v-sheet--tile theme--light"
      role="list"
    >
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-list-item__content"
        >
          <div
            class="v-list-item__title"
          >
            Правила поведения на улице kopnik.org
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            Сходатай на копе
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            
          </div>
        </div>
      </div>
      <div
        class="v-list-item theme--light"
        role="listitem"
        tabindex="-1"
      >
        <div
          class="v-list-item__content"
        >
          <div
            class="v-list-item__title"
          >
            Территориальные чаты kopnik.org
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            Автор идеи/Старший куратор
          </div>
           
          <div
            class="v-list-item__subtitle"
          >
            
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
